h/static/scripts/guest.coffee

```diff
@@ -119,9 +120,27 @@ class Annotator.Guest extends Annotator
       # Announce the new positions, so that the sidebar knows
       this.plugins.Bridge.sync([highlight.annotation])
 
+  # Utility function to remove the hash part from a URL
+  _removeHash: (url) ->
+    url = new URL url
```

Comment:
Added polyfill in https://github.com/hypothesis/h/commit/4f46e9ff88455b651e8e039cbaee6ba589633b79.